src/main/java/io/confluent/connect/jdbc/source/TimestampIncrementingTableQuerier.java

```diff
@@ -110,7 +110,11 @@ protected void createPreparedStatement(Connection db) throws SQLException {
       //  timestamp 1235, id 22
       //  timestamp 1236, id 23
       // We should capture both id = 22 (an update) and id = 23 (a new row)
-      builder.append(" WHERE ");
+      if (query.indexOf("where") != -1) {
```

Comment:
If in given query there is a column name which has 'where' string then below ` if (query.indexOf("where") != -1) {` checks will failed.
e.g. column name could be **whereabouts**
Q. Whereabouts in Madrid do you live?
The 'WHERE' string should prefix and suffix by space i.e. ` if (query.indexOf(" where ") != -1) {`.